.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -4,44 +4,56 @@ about: Suggest an idea
 title: ''
 labels: ''
 assignees: ''
-
 ---
+# Suggestion
 
-<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨
-
-Half of all issues filed here are duplicates, answered in the FAQ, or not appropriate for the bug tracker.
-
-Please help us by doing the following steps before logging an issue:
-  * Search: https://github.com/Microsoft/TypeScript/search?type=Issues
-  * Read the FAQ, especially the "Common Feature Requests" section: https://github.com/Microsoft/TypeScript/wiki/FAQ
-
+<!--
+  Please fill in each section completely. Thank you!
 -->
 
-## Search Terms
+## 🔍 Search Terms
 
-<!-- List of keywords you searched for before creating this issue. Write them down here so that others can find this suggestion more easily -->
+<!--
+  💡 Did you know? TypeScript has over 2,000 open suggestions!
+  🔎 Please search thoroughly before logging new feature requests as most common ideas already have a proposal in progress.
+  The "Common Feature Requests" section of the FAQ lists many popular requests: https://github.com/Microsoft/TypeScript/wiki/FAQ#common-feature-requests
 
-## Suggestion
+  Replace the text below:
+-->
 
-<!-- A summary of what you'd like to see added or changed -->
+List of keywords you searched for before creating this issue. Write them down here so that others can find this suggestion more easily and help provide feedback.
 
-## Use Cases
+## ✅ Viability Checklist
 
 <!--
-What do you want to use this for?
-What shortcomings exist with current approaches?
+   Suggestions that don't meet all these criteria are very, very unlikely to be accepted.
+   We always recommend reviewing the TypeScript design goals before investing time writing
+   a proposal for ideas outside the scope of the project.
 -->
-
-## Examples
-
-<!-- Show how this would be used and what the behavior would be -->
-
-## Checklist
-
 My suggestion meets these guidelines:
 
 * [ ] This wouldn't be a breaking change in existing TypeScript/JavaScript code
 * [ ] This wouldn't change the runtime behavior of existing JavaScript code
 * [ ] This could be implemented without emitting different JS based on the types of the expressions
-* [ ] This isn't a runtime feature (e.g. library functionality, non-ECMAScript syntax with JavaScript output, etc.)
+* [ ] This isn't a runtime feature (e.g. library functionality, non-ECMAScript syntax with JavaScript output, new syntax sugar for JS, etc.)
 * [ ] This feature would agree with the rest of [TypeScript's Design Goals](https://github.com/Microsoft/TypeScript/wiki/TypeScript-Design-Goals).
+
+
+## ⭐ Suggestion
+
+<!-- A summary of what you'd like to see added or changed -->
+
+## 💻 Use Cases
```

Comment:
Is there much of a difference between "Use Cases" and "Motivating Example"?

Reply:
I like having more places for people to provide color